src/Symfony/Component/JsonPath/Tests/JsonCrawlerTest.php

```diff
@@ -658,7 +654,7 @@ public static function provideSingleQuotedStringProvider(): array
                 ['with tab'],
             ],
             [
-                "$['quote\\\"here']",
+                '$["quote\\"here"]',
```

Comment:
this data provider is named `provideSingleQuotedStringProvider`, so it should not be changed to use double-quoted strings IMO

Reply:
Ah yes indeed